security-policies/bundle/compliance/cis_azure/rules/cis_5_1_4/rule.rego

```diff
@@ -0,0 +1,32 @@
+package compliance.cis_azure.rules.cis_5_1_4
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import future.keywords.if
+
+# Ensure the storage account containing the container with activity logs is encrypted with Customer Managed Key
+finding = result if {
+	# filter
+	data_adapter.is_storage_account
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(is_customer_managed_key_encrypted),
+		evidence,
+	)
+}
+
+is_customer_managed_key_encrypted if {
+	data_adapter.resource.extension.storageAccount.properties.encryption.keySource == "Microsoft.KeyVault"
+	common.contains_key(data_adapter.resource.extension.storageAccount.properties.encryption, "keyVaultProperties")
+	data_adapter.resource.extension.storageAccount.properties.encryption.keyVaultProperties != null
+} else = false
+
+evidence = {
```

Comment:
Why isn't the evidence the whole storage account? That's how I've commonly saw it being implemented

Reply:
figured sending the relevant fields for 
- tracking the resource
- reasoning why it failed 

is the better way to go, as the whole resource is already being sent anyway and selecting just those fields makes it easier to tell why the rule failed

i'm open to change this if you think it'd be better

Reply:
Hmmmm just had not seen the evidence being manipulated. Usually I see just the resource being sent. But if you are confident this is the way to go, sounds good to me

Reply:
Working on #1509, I found that we manipulate the `evidence` field in approximately 15% of cases. IMHO as part of #1509 we will need to agree on a common `evidence` strategy.